tox.ini

```diff
@@ -37,6 +37,20 @@ environment =
     MYSQL_ALLOW_EMPTY_PASSWORD=1
     MYSQL_USER=quay
 
+[testenv:py39-unit]
```

Comment:
why do we need this new block?

Reply:
this was after our discussion that we want to add `SKIP_DB_SCHEMA=true` for `py39-unit` and essentially have 3 unit test jobs with different DBs. Did you have a different suggestion?